docs/modules/ROOT/pages/module/code.adoc

```diff
@@ -120,4 +119,436 @@ image::code-settings-advanced.png[Code Advanced Settings]
 
 * *Installing the app outside Defender 2.0*: Code Inspector must be installed via Defender 2.0. If you attempt to install it from elsewhere, the installation will not succeed. Ensure you're logged in to your OpenZeppelin account and navigate to Code Inspector from Defender 2.0 for a successful installation.
 
-* *Code Inspector Access*: Access to Code Inspector is required for a successful installation. If you find that you don't have access and you think this is a mistake, contact OpenZeppelin support to get the necessary permissions.
\ No newline at end of file
+* *Code Inspector Access*: Access to Code Inspector is required for a successful installation. If you find that you don't have access and you think this is a mistake, contact OpenZeppelin support to get the necessary permissions.
+
+[[rules]]
+== Rules
+
+[cols='1,1,1']
+|===
+|ID|Description|Severity
+
+|`alert-uniswap-v2-router-liquidity-considerations`
+|Identifies any instance of a Uniswap Router V2 addLiquidity call.
+|note
+
+|`array-length-to-stack`
+|Identifies when the length of an array can be written to the stack to save gas.
+|note
+
+|`call-with-arbitrary-address-bytes`
+|Identifies a potentially unsafe external call.
+|medium
+
+|`chainlink-deprecated-functions`
+|Identifies usage of chainlink's deprecated functions.
+|medium
+
+|`check-consistent-usage-of-msgsender-msgdata`
+|Identifies usage of `msg.sender` or `msg.data` when `_msgSender()` or/and `_msgData()` are present
+|note
+
+|`check-effect-interact`
+|Identifies a possible violation of the check, effect and interact pattern.
+|ethtrust
+
+|`check-erc4337-compatibility`
+|Identifies if the contract may not be compatible with ERC-4337.
+|note
+
+|`check-return-data-from-external-call`
+|Identifies when the external call return data check is missing.
+|note
+
+|`conflicting-inheritance`
```

Comment:
I see this rule and others are marked as not-defender in the [contract inspector docs](https://github.com/OpenZeppelin/contract-inspector/blob/main/docs/rule_ids.md). Does that mean that they are not included for Defender?

Reply:
thanks! fixed